yubikey.go

```diff
@@ -18,35 +18,65 @@ import (
 	"golang.org/x/crypto/pbkdf2"
 )
 
-func DecryptYubikey(msg *jwe.Message, clevisNode map[string]interface{}) ([]byte, error) {
-	yubikeyNode, ok := clevisNode["yubikey"].(map[string]interface{})
-	if !ok {
-		return nil, fmt.Errorf("clevis.go/yubikey: cannot parse provided token, node 'clevis.yubikey'")
-	}
+// YubikeyPin represents the data yubikey needs to perform decryption
+type YubikeyPin struct {
+	Type      string     `json:"type"`
+	Challenge string     `json:"chalelenge"`
```

Comment:
The json field name needs to be `challenge`, otherwise tests fail.

Reply:
Good catch!  Will fix.